test/TensorFlowRuntime/dynamic_attributes.swift

```diff
@@ -0,0 +1,62 @@
+// RUN: %target-run-dynamic-compilation-swift --attributes-json %S/Inputs/attributes.json
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+// REQUIRES: tensorflow
+
+import TensorFlow
+import TensorFlowUnittest
+import StdlibUnittest
+import Python
+
+var DynamicAttributeTests = TestSuite("DynamicAttribute")
+
+/// Loads an attribute value from a file. This ensures that this test actually
+/// exercises dynamic attributes.
+func loadAttribute<T: PythonConvertible>(_ key: String) -> T {
```

Comment:
Can't we use swift's json processing API?

Reply:
I tried, but then realized that that's in Foundation, which the tests do not have access to. I guess eventually when we move Python out of the stdlib, we won't have access to Python either. At that point, we'll have to use glibc or find some other way to make things be certainly dynamic.